### PR TITLE
Release/0.8.3

### PR DESCRIPTION
**Pull request by foreleven**
* fix: factory stated bean's metadata
* chore: upgrade all dependencies

### CODE REVIEW

**Comment by JounQin** [comment]
Need rebase

File: example/src/components/Counter/index.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ export interface CounterProps {
 // };
 
 export function Counter(props: CounterProps) {
-  const counter = useBean(CounterModel, { props });
+  const counter = useBean(() => new CounterModel(), { props });
```
Comment (JounQin) [inline example/src/components/Counter/index.tsx]: Any difference?

**Comment by JounQin** [comment]
The testing is still failing, maybe bump version is required.
`yarn upgrade --latest`

File: example/src/components/Counter/index.tsx (hunk 1 above)
Comment (foreleven) [reply example/src/components/Counter/index.tsx]: a factory method to create the bean for passing some object from react context into stated bean

Comment (JounQin) [reply example/src/components/Counter/index.tsx]: I still can't get the point, is the previous code not working as expected as current one now? I supposed they were just exactly same.

**Comment by JounQin** [comment]
The bundle size is more than 4KB nowx

**Pull request merged**